bot/code_review_bot/__init__.py

```diff
@@ -127,6 +127,19 @@ def build_extra_identifiers(self):
         """
         return {}
 
+    @property
+    def allow_before_and_after_publish(self):
+        """
+        Allow the possibility for an issue to avoid being published based on before/after.
+        This allow publishing issues based on other criteria, like in_patch.
+        """
+        if taskcluster.secrets.get(
+            f"{self.display_name.upper()}_DISABLE_PUBLICATION_BEFORE_AFTER", False
```

Comment:
```suggestion
            f"{self.analyzer.name.upper()}_DISABLE_PUBLICATION_BEFORE_AFTER", False
```
In the code `issue.name` == `issue.analyzer.display_name` == `issue.analyzer.name`, but we should rather pick the original name of the analyzer here (from the task definition).